lib/config/configuration.go

```diff
@@ -195,7 +195,7 @@ func ReadConfigFile(cliConfigPath string) (*FileConfig, error) {
 	if cliConfigPath != "" {
 		configFilePath = cliConfigPath
 		if !utils.FileExists(configFilePath) {
-			return nil, trace.NotFound("file %s is not found", configFilePath)
+			return nil, trace.NotFound("failed to load configuration: configuration file specified '%s' does not exist", configFilePath)
```

Comment:
I think we can still simplify this. We don't need to say `configuration file` twice
```suggestion
			return nil, trace.NotFound("failed to load configuration: file %q does not exist", configFilePath)
```